packages/node_modules/@cerebral/http/src/request.js

```diff
@@ -1,11 +1,16 @@
 /* eslint-env browser */
 
-export default function request(options, cb) {
+export default function request(options = {}, cb) {
   const xhr = new XMLHttpRequest()
+
   xhr.addEventListener('progress', cb)
   xhr.addEventListener('load', cb)
   xhr.addEventListener('error', cb)
   xhr.addEventListener('abort', cb)
+
+  xhr.ontimeout = cb.bind(null, { type: 'timeout', currentTarget: xhr })
```

Comment:
There is a [eslint rule](https://eslint.org/docs/rules/callback-return) that forbids me from calling the callback with only one parameter. I can either keep this, or change the name of the variable.

On this same topic: I'm not 100% sure that this was in the scope of the initial issue, but it would be a nice to have.

Reply:
I don't think we use this rule in Cerebral as it's not part of standard.

Reply:
Oh... my bad, wrong link... I do get this 
> [eslint] Unexpected literal in error position of callback. (standard/no-callback-literal)
when trying to use it like:
```js
xhr.ontimeout = function() {
  cb({ type: 'timeout', currentTarget: xhr })
}
```
I've done further reading and fixed it

Reply:
Great!